src/hyperlight_host/src/sandbox/initialized_multi_use.rs

```diff
@@ -611,10 +594,6 @@ impl MultiUseSandbox {
         if self.poisoned {
             return Err(crate::HyperlightError::PoisonedSandbox);
         }
-        // Mark that a guest function call is now active
-        // (This also increments the generation counter internally)
-        // The guard will automatically clear call_active when dropped
-        let _guard = CallActiveGuard::new(self.vm.interrupt_handle())?;
```

Comment:
Where did this go to?  Is call_active still being used?  What are the other changes related to this?

Reply:
This PR no longer changes cancellation logic